frontend/src/components/taskSelection/tests/taskList.test.js

```diff
@@ -39,43 +38,37 @@ describe('Task Item', () => {
 
   it('should set the zoom task ID of the task to be zoomed', async () => {
     const setZoomedTaskIdMock = jest.fn();
-    renderWithRouter(
+    const { user } = renderWithRouter(
       <IntlProviders>
         <TaskItem data={task} selectTask={jest.fn()} setZoomedTaskId={setZoomedTaskIdMock} />
       </IntlProviders>,
     );
-    await userEvent.click(screen.getAllByRole('button')[1]);
+    await user.click(screen.getAllByRole('button')[1]);
     expect(setZoomedTaskIdMock).toHaveBeenCalledWith(8);
   });
 
   it('should copy task URL to the clipboard', async () => {
-    const originalClipboard = { ...global.navigator.clipboard };
-    const mockClipboard = {
-      writeText: jest.fn().mockImplementation(() => Promise.resolve()),
-    };
-    global.navigator.clipboard = mockClipboard;
-    createComponentWithMemoryRouter(
+    const { user } = createComponentWithMemoryRouter(
```

Comment:
Is `{ copyToClipboard: true }` not required for this test?

Reply:
Apparently not. Or at least it wasn't failing for me.